code/modules/admin/view_variables/topic.dm

```diff
@@ -0,0 +1,142 @@
+//DO NOT ADD MORE TO THIS FILE.
+//Use vv_do_topic() for datums!
+/client/proc/view_var_Topic(href, href_list, hsrc)
+	if( (usr.client != src) || !src.holder || !holder.CheckAdminHref(href, href_list))
+		return
+	var/target = GET_VV_TARGET
+	vv_do_basic(target, href_list, href)
+	if(istype(target, /datum))
+		var/datum/D = target
+		D.vv_do_topic(href_list)
+	else if(islist(target))
+		vv_do_list(target, href_list)
+	if(href_list["Vars"])
+		debug_variables(locate(href_list["Vars"]))
+
+	//rest of this should proabbly be eventually moved to vv_do_topic in datums
+
+
+
+
+
+
+
+
+
+
+
+
+//Needs +VAREDIT past this point
```

Comment:
If yer wondering why this is still in it's because they weren't dropdown options, I can convert 'em to use vv_do_topic too if needed.